Readme.md

```diff
@@ -12,51 +10,52 @@ for a guide to upgrading from 1.x to 2.x.
 
 -----
 
-### Canvas graphics API backed by Cairo
 [![Build Status](https://travis-ci.org/Automattic/node-canvas.svg?branch=master)](https://travis-ci.org/Automattic/node-canvas)
 [![NPM version](https://badge.fury.io/js/canvas.svg)](http://badge.fury.io/js/canvas)
 
-  node-canvas is a [Cairo](http://cairographics.org/) backed Canvas implementation for [NodeJS](http://nodejs.org).
-
-## Authors
-
-  - TJ Holowaychuk ([tj](http://github.com/tj))
-  - Nathan Rajlich ([TooTallNate](http://github.com/TooTallNate))
-  - Rod Vagg ([rvagg](http://github.com/rvagg))
-  - Juriy Zaytsev ([kangax](http://github.com/kangax))
+node-canvas is a [Cairo](http://cairographics.org/)-backed Canvas implementation
+for [Node.js](http://nodejs.org).
 
 ## Installation
 
 ```bash
 $ npm install canvas
 ```
 
-By default, binaries for macOS, Linux and Windows will be downloaded. If you want to build from source, use `npm install --build-from-source`.
+By default, binaries for macOS, Linux and Windows will be downloaded. If you
+want to build from source, use `npm install --build-from-source` and see the
+**Compiling** section below.
 
-Currently the minimum version of node required is __6.0.0__
+The minimum version of Node.js required is __6.0.0__.
 
 ### Compiling
 
-If you don't have a supported OS or processor architecture, or you use `--build-from-source`, the module will be compiled on your system. Unless previously installed you'll _need_ __Cairo__ and __Pango__. For system-specific installation view the [Wiki](https://github.com/Automattic/node-canvas/wiki/_pages).
+If you don't have a supported OS or processor architecture, or you use
+`--build-from-source`, the module will be compiled on your system. This requires
+several dependencies, including Cairo and Pango.
 
-You can quickly install the dependencies by using the command for your OS:
+For detailed installation information, see the
+[wiki](https://github.com/Automattic/node-canvas/wiki/_pages). One-line
+installation instructions for common OSes are below. Note that libgif/giflib,
+librsvg and libjpeg are optional and only required if you need GIF, SVG and JPEG
+support, respectively.
 
 OS | Command
 ----- | -----
-OS X | Using [Homebrew](https://brew.sh/):<br/>`brew install pkg-config cairo pango libpng jpeg giflib`<br/><br/>Using [MacPorts](https://www.macports.org/):<br/>`port install pkgconfig cairo pango libpng jpeg giflib libsrvg`
-Ubuntu | `sudo apt-get install libcairo2-dev libjpeg-dev libpango1.0-dev libgif-dev build-essential g++`
-Fedora | `sudo yum install cairo cairo-devel cairomm-devel libjpeg-turbo-devel pango pango-devel pangomm pangomm-devel giflib-devel`
+OS X | Using [Homebrew](https://brew.sh/):<br/>`brew install pkg-config cairo pango libpng jpeg giflib librsvg`<br/><br/>Using [MacPorts](https://www.macports.org/):<br/>`port install pkgconfig cairo pango libpng jpeg giflib librsvg`
```

Comment:
Maybe it's time to remove "MacPorts"? Is there anyone that is still using that? 💭

Reply:
I'll put it in the wiki and remove it from here.